core/sass/components/_form-wizard.scss

```diff
@@ -85,6 +85,10 @@
       color: white;
     }
   }
+  #triage-company-name-not-in-companies-house {
```

Comment:
since we removed the original triage can we change these to something like `contact-triage-` instead? just to be explicit (doesn't have to be in this PR if there are a lot of them to change)